packs/data/consumable-effects.db/effect-lastwall-soup-greater.json

```diff
@@ -0,0 +1,55 @@
+{
+    "_id": "neZPoQF4hW3A31dd",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @Compendium[pf2e.feats-srd.Gravelands Herbalist]{Gravelands Herbalist}</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth checks against undead</li>\n<li>10 temporary hit points</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>+4 item bonus to Stealth DCs against undead</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Lastwall Soup (vs. undead)",
+                "predicate": {
+                    "all": [
+                        "undead"
+                    ]
+                },
+                "selector": "stealth",
+                "type": "item",
+                "value": 4
+            },
+            {
+                "key": "TempHP",
+                "value": 10
+            }
+        ],
+        "source": {
+            "value": ""
```

Comment:
```suggestion
            "value": "Pathfinder Lost Omens: Knights of Lastwall"
```

Reply:
Fixed in [6ea844f](https://github.com/foundryvtt/pf2e/pull/2596/commits/6ea844f1a79caa3ce2cafba0b28c385303c78d53)